package.json

```diff
@@ -53,5 +53,11 @@
     "postcss-styled-syntax": "^0.4.0",
     "stylelint": "^15.10.2",
     "stylelint-config-standard": "^34.0.0"
+  },
+  "pnpm": {
+    "overrides": {
+      "@types/eslint": "8.4.1",
+      "typescript": "4.6.3"
+    }
```

Comment:
The package upgrades created some linting chaos. Specifically, some dependency started pulling a different conflicting version of `@types/eslint` and another dependency started pulling typescript 5, both of which created type collisions that made linting fail. One way to deal with this is with pnpm package overrides like this, which force all versions of a package in the dependency array to have the same version.

Reply:
Nice, glad this was easy to fix